README.md

```diff
@@ -277,9 +277,8 @@ to intervene to uphold that code of conduct.
 
 ## License
 
-The contents of this repository are dual-licensed under the _MIT OR Apache
-2.0_ License. That means you can chose either the MIT licence or the
-Apache-2.0 licence when you re-use this code. See `MIT` or `APACHE2.0` for more
+The contents of this repository are dual-licensed under the _[MIT](LICENSE-MIT) OR [Apache-2.0](LICENSE-APACHE_2.0)_ License. That means you can chose either the MIT licence or the
```

Comment:
This first link to Apache 2.0 license has a typo (_ instead of -)
Did you test it?